src/tctrack/tempest_extremes/tempest_extremes.py

```diff
@@ -373,6 +373,12 @@ class DetectNodesParameters:
     search_by_min field or highest value of the search_by_max field are retained.
     """
 
+    time_filter: str | None = None
+    """
+    Filter for the input data frequency. Options are: `"3hr"`, `"6hr"`, `"daily"`.
+    Alternatively, can be a regex for the datetime in the format `YYYY-MM-DD HH:MM:SS`.
```

Comment:
Should the regex also be a string?
```suggestion
    Alternatively, can be a regex for the datetime in the format `"YYYY-MM-DD HH:MM:SS"`.
```